grails-app/assets/javascripts/meriplan.js

```diff
@@ -720,6 +723,39 @@ function MERIPlan(project, projectService, config) {
         }
         return canApprove
     };
+
+}
+function validateFloristics(field) {
+
+    var ctx  = ko.dataFor(field);
```

Comment:
The reason ctx is null/undefined here is jqueryValidationEngine passes a jquery object wrapping the field, not the element itself.
ko.dataFor(field[0]) will return the view model for the row.